### PR TITLE
Unbundle govuk-frontend

### PR DESCRIPTION
**Pull request by domoscargin**
https://trello.com/c/fhL8Zl8T/175-3-unbundle-govuk-frontend-from-digitalmarketplace-govuk-frontend

## Why
We want to be able to use digitalmarketplace-govuk-frontend with [LandRegistry/govuk-frontend-jinja](https://github.com/LandRegistry/govuk-frontend-jinja) and reduce the coupling with [govuk-frontend](https://github.com/alphagov/govuk-frontend).

## Testing
Local tests and the app are working.

There is a PR for the Briefs Frontend for an example of how to get a frontend app working with this change: https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/352

### CODE REVIEW

File: src/digitalmarketplace/components/alert/_alert.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@import "govuk/tools/all";
-@import "govuk/helpers/all";
+@import "node_modules/govuk-frontend/tools/all";
```
Comment (lfdebrux) [inline src/digitalmarketplace/components/alert/_alert.scss:1]: As we're copying govuk-frontend to `src` and adding `src` to the Sass `includePaths` we shouldn't need to change these paths right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ See below for Changelog examples.
 
 💥 Breaking changes:
 
+  - GOV.UK Frontend has been removed from the package.
+    - You will need to install GOV.UK Frontend separately - we recommend using (npm)](docs/installation/installing-with-npm.md).
+    - You should ensure that GOV.UK Frontend templates are loaded into your templating environment. If you are using v3, including `node_modules/govuk-frontend` should be sufficient. If you are using v2, you will need to prefix the templates with `govuk` to ensure they are correctly located.
```
Comment (lfdebrux) [inline CHANGELOG.md:11]: 👍

File: src/digitalmarketplace/components/alert/_alert.scss (hunk 1 above)
Comment (domoscargin) [reply src/digitalmarketplace/components/alert/_alert.scss]: Ah, correct - this is a hangover from before I knew about prefixLoader for Briefs

File: tasks/gulp/clean.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ src.displayName = 'clean:src'
 src.description = 'Cleans `govuk` folder from src'
 
 const pkg = async (done) => {
-  await cleanupFoldersOrFiles(['package/govuk', 'package/govuk-frontend', 'package/digitalmarketplace'])
+  await cleanupFoldersOrFiles('package/digitalmarketplace')
```
Comment (lfdebrux) [inline tasks/gulp/clean.js:29]: Can we have this still delete `package/govuk` to avoid devs like me making silly mistakes? 😁

Comment (domoscargin) [reply tasks/gulp/clean.js]: We can and we should!

**Review by lfdebrux — APPROVED** [review]
This looks good to me!

File: CHANGELOG.md (hunk 2 above)
Comment (lfdebrux) [reply CHANGELOG.md:11]: We'll also need to make the govuk-frontend stylesheets available at `govuk/` for Sass to compile, we should probably mention that here too.

**Pull request merged**